src/components/ProvisionSmartWalletNoticeDialog.tsx

```diff
@@ -12,14 +12,19 @@ const useSmartWalletFeeQuery = (rpc: string | null) => {
     const fetchParams = async () => {
       assert(rpc);
       try {
-        const params = await querySwingsetParams(rpc);
+        const client = await agoric.ClientFactory.createRPCQueryClient({
```

Comment:
Remark: the wallet connection also makes its own query client, ideally at some point we could share between these two places somehow. I think the only main downside currently is an extra "status" query when you create an extra client.